scripts/generate-plugins/main.py

```diff
@@ -30,14 +30,17 @@
 
 
 PLUGIN_DISPLAY_TYPES = {
+  'source': 'Source',
```

Comment:
these shouldn't be needed. Where do you see these? Only batchsource and streamingsource are supported source types in CDAP.

Reply:
I got this value during plugin generation as `plugin_type`. Thought that this was a valid type

Reply:
Could you tell me which plugins emit these as types?

Reply:
Kudu and Kafka

Reply:
Are these the ones?
![image](https://user-images.githubusercontent.com/3895099/56222074-3bccac80-6020-11e9-8d45-218c4652f3ff.png)


If so, once you add them to the excludes list, you shouldn't see them, right?

Reply:
Yes, correct

Reply:
Do I need to exclude them?

Reply:
yes, all these should be excluded, and so you won't need any of the changes to that plugin types dictionary above.

Reply:
done